controls/radgauge/getting-started/creating-radial-gauge.md

```diff
@@ -77,20 +69,19 @@ The __RadialScale__ control allows you to display different types of indicators
 	</telerik:RadRadialGauge>
 {{endregion}}
 
-Here is a snapshot of the result.
+#### __Image 2: Result from Example 2__
 
-![](images/RadGauge_GettingStarted_RadialGauge_03.png)
+![RadRadialGauge with indicator](images/RadGauge_GettingStarted_RadialGauge_03.png)
 
 ## Set Indicator's Position
 
 To make the indicator point to a certain value from the scale, you have to use the __Value__ property of the indicator. Here is an example:        

```

Comment:
Instead of "Here is an example:" use something like "Example 3 shows how you can work with this property".